scripts/components/reportsDashboards/build.sh

```diff
@@ -85,6 +85,10 @@ esac
 
 CHROMIUM_URL="https://github.com/opensearch-project/dashboards-reports/releases/download/chromium-1.12.0.0/$CHROMIUM_TARGET"
 
+MINOR_VERSION=${VERSION%.*}
+git clone --branch $MINOR_VERSION --single-branch https://github.com/opensearch-project/OpenSearch-Dashboards ../../OpenSearch-Dashboards || echo repo exists
```

Comment:
so we are downloading Dashboards if it doesn't exist so it can build the dashboards reports component?

Reply:
yes

Reply:
note that to fix #606 property you would want to also check not just whether that folder exists, but that it was built